### PR TITLE
Various minor fixes

### PR DESCRIPTION
**Pull request by erikdubbelboer**
Too small for a pull each.

### CODE REVIEW

File: features/support/steps/backend.ts [hunk 1]
```diff
@@ -25,6 +25,9 @@ Given('the {string} backend is running', async function (this: World, backend: s
     prc.stderr.on('data', (data: string) => {
       const lines = data.split('\n')
       lines.forEach(line => {
+        if (line === '') {
+          return
+        }
```
Comment (erikdubbelboer) [inline features/support/steps/backend.ts:30]: Don't print empty lines in our logs.

File: features/support/steps/network.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ Given('{string} are joined in a lobby', async function (this: World, playerNames
     const playerName = playerNames[i]
     const player = this.players.get(playerName)
     if (player == null) {
-      return new Error(`player ${playerName} not found`)
+      throw new Error(`player ${playerName} not found`)
```
Comment (erikdubbelboer) [inline features/support/steps/network.ts:37]: Replace a bunch of returns with the correct `throw` we use everywhere.

File: features/support/world.ts [hunk 3]
```diff
@@ -38,10 +38,9 @@ export class World extends CucumberWorld {
 
   public print (message: string): void {
     if (this.scenarioRunning) {
-      this.log(message) as any
-      // console.log(message)
+      void this.attach(message)
     } else {
-      // console.log(message)
+      // this.attach(message)
```
Comment (erikdubbelboer) [inline features/support/world.ts:43]: `this.log` just prints `Attachment (text/x.cucumber.log+plain)` as the cucumber printer for some reason doesn't know about the cucumber log mime type. `this.attach` actually prints the string in the log instead.

File: lib/types.ts [hunk 4]
```diff
@@ -94,7 +94,6 @@ export interface JoinPacket extends Base {
 export interface JoinedPacket extends Base {
   type: 'joined'
   lobby: string
-  id: string
```
Comment (erikdubbelboer) [inline lib/types.ts:97]: Not being used.

File: package.json [hunk 5]
```diff
@@ -32,7 +32,7 @@
   "scripts": {
     "prepare": "yarn build",
     "build": "parcel build",
-    "lint": "ts-standard",
+    "lint": "ts-standard --fix",
```
Comment (erikdubbelboer) [inline package.json:35]: Automatically fix what can automatically be fixed.

**Review by koenbollen — APPROVED** [review]

**Pull request merged**